uxarray/grid/neighbors.py

```diff
@@ -781,6 +781,207 @@ def coordinates(self, value):
             )
 
 
+class SpatialHash:
+    """Custom data structure that is used for performing grid searches using Spatial Hashing. This class constructs an overlying
+    uniformly spaced structured grid, called the "hash grid" on top an unstructured grid. Faces in the unstructured grid are related
+    to the cells in the hash grid by determining the hash cells the bounding box of the unstructured face cells overlap with.
+
+    Parameters
+    ----------
+    grid : ux.Grid
+        Source grid used to construct the hash grid and hash table
+    reconstruct : bool, default=False
+        If true, reconstructs the tree
+
+    Notes
+    -----
+
```

Comment:
Can remove if we don't plan to add anything to here.

Reply:
Resolved in https://github.com/UXARRAY/uxarray/pull/1169/commits/d599276601dc2fedca6b9069070bc52702d579ac